src/services/QRService.ts

```diff
@@ -171,8 +170,8 @@ export class QRService {
    */
   public decryptTrasactionQRText(qrTransactionText: QRTransactionText): TransferTransaction {
```

Comment:
So this function says "decrypt" but it looks like its doing a lot more than decrypting. And i'm not sure i see where it actually IS decrypting anything.

Reply:
their code